### PR TITLE
Update synthetic tests

### LINKED ISSUE #42: Browser Synthetics Test options missing
Found a bug? Maybe our [Slack Community](https://slack.cloudposse.com) can help. 

[Slack Community](https://slack.cloudposse.com)

## Describe the Bug
Browser synthetic tests have `device_ids` and `browser_steps` variables to specify what devices to run the tests from. It's not possible to specify that with this module currently.

https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/synthetics_test#:~:text=resource%20%22datadog_synthetics_test%22%20%22test_browser%22%20%7B%20

## Expected Behavior
I should be able to pass in device_ids and browster_steps for a browser test

## Steps to Reproduce
2. Run '....'
3. Enter '....'

## Screenshots
If applicable, add screenshots or logs to help explain your problem.

## Environment (please complete the following information):

Anything that will help us triage the bug will help. Here are some ideas:
 - OS: [e.g. Linux, OSX, WSL, etc]
 - Version [e.g. 10.15]

### PR DESCRIPTION
**Pull request by aknysh**
## what
* Update synthetic tests
* Add new Datadog monitors
* Fix k8s monitors

## why
* Implement all features and attributes described in https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/synthetics_test 
* New Datadog monitors to monitor RabbitMQ

## related
* Closes #42

### CODE REVIEW

File: catalog/monitors/k8s.yaml [hunk 1]
```diff
@@ -246,14 +246,14 @@ k8s-node-status-unschedulable:
     #warning_recovery:
 
 k8s-imagepullbackoff:
-  name: "(k8s) ImagePullBackOff detected"
-  type: event alert
+  name: "(k8s) dev - ImagePullBackOff detected"
```
Comment (osterman) [inline catalog/monitors/k8s.yaml:249]: why hardcode dev?

File: catalog/monitors/rabbitmq.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  type: "query alert"
+  query: |
+    avg(last_5m):avg:rabbitmq.node.mem_used{*} by {host} / avg:system.mem.total{*} by {host} * 100 > 35
+  message: |
+    RabbitMQ is using too many resources on host: {{host.name}}.
+    It may block connections and won't be able to perform many internal operations.
+  escalation_message: ""
+  tags:
+    - "managed-by:Terraform"
+    - "integration:rabbitmq"
```
Comment (osterman) [inline catalog/monitors/rabbitmq.yaml:47]: case inconsistency

File: examples/synthetics/synthetics/http.yaml [hunk 3]
```diff
@@ -1,50 +1,60 @@
 simple:
-  name: cloudposse.com simple test
-  message: CloudPosse simple is not reachable
+  name: "cloudposse.com simple test"
+  message: "CloudPosse simple is not reachable"
```
Comment (osterman) [inline examples/synthetics/synthetics/http.yaml:3]:
```suggestion
  name: "cloudposse.com website test"
  message: "Cloud Posse website is not reachable"
```

File: examples/synthetics/synthetics/http.yaml [hunk 4]
```diff
@@ -1,50 +1,60 @@
...
+  assertion:
     - type: statusCode
       operator: is
       target: "200"
 
 complex:
-  name: cloudposse.com complex test
-  message: CloudPosse complex is not reachable
+  name: "cloudposse.com complex test"
+  message: "CloudPosse complex is not reachable"
```
Comment (osterman) [inline examples/synthetics/synthetics/http.yaml:26]:
```suggestion
  message: "Cloud Posse website is not reachable (advanced test)"
```

File: catalog/monitors/k8s.yaml (hunk 1 above)
Comment (aknysh) [reply catalog/monitors/k8s.yaml:249]: copy-paste from another place

**Review by osterman — APPROVED** [review]

**Pull request merged**